### PR TITLE
[no ticket][risk=no] Puppeteer: Combine R notebook tests to shorten test time

### PR DESCRIPTION
**Pull request by aweng98**
`import-r-lib.spec.ts` test steps combined with `notebook-r.spec.ts` test. One less notebook runtime startup helps to shorten test suite playback time.
Notebook test coverage is unchanged.

### CODE REVIEW

**Comment by aweng98** [comment]
@NehaBroad Hi, I would be grateful for your prompt review. CircleCI job time is insufferable and painfully long.

File: e2e/app/page/notebook-page.ts [hunk 1]
```diff
@@ -222,13 +222,11 @@ export default class NotebookPage extends AuthenticatedPage {
     if (mode === Mode.Command) {
       await this.page.keyboard.press('Escape');
       await this.getIFrame().then(frame => frame.waitForSelector('body.notebook_app.command_mode'));
-      console.log('Notebook in command mode');
```
Comment (aweng98) [inline e2e/app/page/notebook-page.ts:225]: not useful information.

File: e2e/app/page/notebook-page.ts [hunk 2]
```diff
@@ -222,13 +222,11 @@ export default class NotebookPage extends AuthenticatedPage {
...
       return;
     }
     // Press Enter key to activate edit mode
     await this.page.keyboard.press('Enter');
     await this.getIFrame().then(frame => frame.waitForSelector('body.notebook_app.edit_mode'));
-    console.log('Notebook in edit mode');
```
Comment (aweng98) [inline e2e/app/page/notebook-page.ts:231]: not useful information.

File: e2e/tests/notebook/notebook-r.spec.ts [hunk 3]
```diff
@@ -23,14 +25,32 @@ describe('Jupyter notebook tests in R language', () => {
...
+    const codeCell = await notebook.findCell(cellIndex);
+    await codeCell.focus();
+    await notebook.changeToMarkdownCell();
+    const markdownCell = await notebook.findCell(cellIndex, CellType.Markdown);
+    const markdownCellInput = await markdownCell.focus();
+    await markdownCellInput.type(rCodeSnippet);
+    await notebook.changeToCodeCell();
+
+    const cell3Output = await notebook.runCodeCell(cellIndex, {timeOut: 5 * 60 * 1000});
+    expect(cell3Output).toContain('success');
```
Comment (aweng98) [inline e2e/tests/notebook/notebook-r.spec.ts:52]: Copy from `import-r-lib.spec.ts` test.

**Review by calbach — APPROVED** [review]

**Pull request merged**